src/api/atlas_sdk_client.js

```diff
@@ -468,15 +468,25 @@ class AtlasSDKClient {
    *    }
    * });
    */
-  getRecommendations(sku, options = {
-    reco_id: '',
-    tracking_string: '',
-    type: ''
-  }) {
+  getRecommendations(sku, options) {
```

Comment:
Could we update the documentation of the `options` input to this method to include `type` and  `filters` as well?

Reply:
Done

Reply:
Thanks for adding the description for `filters`. FYI: I also added the docs for `type`